accounts/iam_workflow_support.tf

```diff
@@ -93,4 +93,33 @@ data "aws_iam_policy_document" "workflow_support" {
       "*",
     ]
   }
+
+  # Allow workflow-support users to browse the contents of the storage service,
+  # and download files in the digitised/ prefix.
+  #
+  # This is used to copy videos into the Goobi storage bucket for reingest.
+  # See https://wellcome.slack.com/archives/CN56BRQ5B/p1594804248033200
```

Comment:
Worth summarising this in the ticket?

Reply:
I'm hoping there's a sufficient summary is in the comment; the link is just to tie it all together.